src/ophys_etl/modules/suite2p_registration/schemas.py

```diff
@@ -86,6 +86,14 @@ def setup_default_suite2p_args(self, data: dict, **kwargs) -> dict:
         data['suite2p_args']['do_registration'] = 1
         data['suite2p_args']['reg_tif'] = True
         data['suite2p_args']['retain_files'] = ["*.tif", "ops.npy"]
+
+        if "nimg_init" not in data["suite2p_args"]:
```

Comment:
these 6 lines are not necessary, because you are already defaulting them in the `Suite2PWrapperSchema`